front/src/App.vue

```diff
@@ -1,20 +1,21 @@
 <template>
-  <div>
-    <Map />
-  </div>
+  <v-app>
+    <doran-app-bar />
+    <router-view></router-view>
+  </v-app>
 </template>
 
 <script>
-import Map from "./components/Map";
+import doranAppBar from "./components/DoranAppBar";
 
 export default {
   name: "app",
   components: {
-    Map,
+    doranAppBar,
```

Comment:
```suggestion
    DoranAppBar,
```
개인적으로 Vue의 Component는 Java의 Class로 봤으면 좋겠어요.
외부의 클래스를 사용하는 것은 대문자로 쓰지요.

사실 이 부분은 정확하게 컨벤션이 있는 건 아닌 것 같아요. 😓 
하지만 [스타일 가이드](https://kr.vuejs.org/v2/style-guide/index.html#JS-JSX%EC%97%90%EC%84%9C-%EC%BB%B4%ED%8F%AC%EB%84%8C%ED%8A%B8-%EC%9D%B4%EB%A6%84-%EA%B7%9C%EC%B9%99-%EC%A7%80%EC%A0%95-casing-%EB%A7%A4%EC%9A%B0-%EC%B6%94%EC%B2%9C%ED%95%A8)를 읽다 보면 전역 컴포넌트의 스크립트 단 네이밍 호출은 파스칼 케이스를 추천하고 있네요.